pkg/vulnerability/harbor.go

```diff
@@ -0,0 +1,231 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package vulnerability
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+	"go.uber.org/zap"
+)
+
+const (
+	username                 = "DOCKER_USERNAME"
+	password                 = "DOCKER_PASSWORD"
+	vulnerabilityHeader      = "X-Accept-Vulnerabilities"
+	vulnerabilityHeaderValue = "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
+	skipTLSVerify            = "SKIP_TLS"
+	certificate              = "CERTIFICATE_PATH"
```

Comment:
Let's rename this to HARBOR_REGISTRY_CACERT

Reply:
Also, would it be possible to accept an optional HARBOR_REGISTRY_CAPATH variable? This variable would allow users to point to the path of the directory containing the CA certificates.

Reply:
how we are going to use this